tests/Makefile

```diff
@@ -13,6 +13,8 @@ GINKGO_LABEL=
 GINKGO_FLAGS=
 NGF_VERSION=
 CI=false
+TELEMETRY_ENDPOINT=
+TELEMETRY_ENDPOINT_INSECURE=
```

Comment:
It seems we don't need use the disable-telemetry flag then for our tests (both functional and conformance) as long as we just build with the endpoint var set to empty, right? In that case we'll just log.

Reply:
> It seems we don't need use the disable-telemetry flag then for our tests (both functional and conformance) as long as we just build with the endpoint var set to empty, right? In that case we'll just log.

yep